create_science_data/Create_example_level1_wfi_image.ipynb

```diff
@@ -0,0 +1,789 @@
+    "print(f'asdf: {asdf.__version__}')\n",
+    "print(f'astropy: {astropy.__version__}')\n",
+    "print(f'numpy: {np.__version__}')\n",
+    "print(f'roman_datamodels: {roman_datamodels.__version__}')"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "5d05f5ae",
+   "metadata": {},
+   "source": [
+    "## Create an empty Level 1 datamodel\n",
+    "\n",
+    "The Roman data models will expect specific data and metadata, these can be seen and referenced in the datamodels and schemas:\n",
+    "\n",
+    "* [ScienceRawModel](https://github.com/spacetelescope/roman_datamodels/blob/main/src/roman_datamodels/datamodels.py#L290) is used as input to the science calibration pipeline, and represents a Level 1 file. With its associated [schema](https://github.com/spacetelescope/rad/blob/main/src/rad/resources/schemas/wfi_science_raw-1.0.0.yaml)\n",
+    "\n",
+    "* [ImageModel](https://github.com/spacetelescope/roman_datamodels/blob/main/src/roman_datamodels/datamodels.py#L286) will contain the collapsed and calibrated ramp for both imaging and spectroscopic modes, and represents a Level 2 file. With its associated [schema](https://github.com/spacetelescope/rad/blob/main/src/rad/resources/schemas/wfi_image-1.0.0.yaml)\n",
+    "\n",
+    "The level 1 datamodel expects that certain metadata are saved along with the data arrays. These can be seen referenced in the schema file for the Level 1 datamodel:\n",
+    "<pre>\n",
+    "%YAML 1.1\n",
+    "---\n",
+    "$schema: asdf://stsci.edu/datamodels/roman/schemas/rad_schema-1.0.0\n",
+    "id: asdf://stsci.edu/datamodels/roman/schemas/wfi_science_raw-1.0.0\n",
+    "\n",
+    "title: |\n",
+    "  The schema for WFI science data (both imaging and spectrographic)\n",
+    "type: object\n",
+    "properties:\n",
+    "  meta:\n",
+    "    allOf:\n",
+    "      - $ref: common-1.0.0  <---- these are the expected metadata\n",
+    "  data:\n",
+    "    tag: tag:stsci.edu:asdf/core/ndarray-1.0.0\n",
+    "    datatype: uint16\n",
+    "    ndim: 3\n",
+    "propertyOrder: [meta, data]\n",
+    "flowStyle: block\n",
+    "required: [meta, data]\n",
+    "</pre>\n",
+    "\n",
+    "That \"common\" [metadata file](https://github.com/spacetelescope/rad/blob/main/src/rad/resources/schemas/common-1.0.0.yaml) contains the list of items and their types, and can reference schemas for each item as well, such as:\n",
+    "<pre>\n",
+    "https://github.com/spacetelescope/rad/blob/main/src/rad/resources/schemas/exposure_type-1.0.0.yaml\n",
+    "\n",
+    "which shows these included items:\n",
+    "\n",
+    "%YAML 1.1\n",
+    "---\n",
+    "$schema: asdf://stsci.edu/datamodels/roman/schemas/rad_schema-1.0.0\n",
+    "id: asdf://stsci.edu/datamodels/roman/schemas/exposure_type-1.0.0\n",
+    "\n",
+    "# Helper file to enumerate viewing modes for exposure and ref_exposure\n",
+    "\n",
+    "type: string\n",
+    "title: Type of data in the exposure (viewing mode)\n",
+    "enum:\n",
```

Comment:
Might be helpful to write that the enum for exposure.type in the schema example shows (if present) the list of acceptable values a keyword can have.

Reply:
done